composer.json

```diff
@@ -10,22 +10,21 @@
     ],
     "homepage": "https://github.com/laravel-notification-channels/africastalking",
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Osaigbovo Emmanuel",
-            "email": "osaigbovoemmanuel1@gmail.com",
-            "role": "Developer"
-        }
-    ],
+    "authors": [{
+        "name": "Osaigbovo Emmanuel",
+        "email": "osaigbovoemmanuel1@gmail.com",
+        "role": "Developer"
+    }],
     "require": {
         "php": "^7.2.5",
-        "africastalking/africastalking": "^2.3",
-        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0",
-        "illuminate/support": "~5.5 || ~6.0 || ~7.0"
+        "appslabke/africastalking": "^2.5.1",
+        "guzzlehttp/guzzle": "^7.0.1",
+        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0 || ^8.0",
```

Comment:
Can drop 5.x now

Reply:
(and same below)

Reply:
I am sure some people are still using 5.x

Reply:
It's out of support (both security and bug fix) but ok to leave it.